### ISSUE TITLE
medborgs don't have defib paddles

### ISSUE THREAD

**Issue by ShadowDeath6** [yogstation13/yogstation#237]
title
end my life and bring me back with dem defibs
@X-TheDark

**Comment by X-TheDark** [comment]
Oh, right.../tg/ gave them exclusively to Syndicate Medborgs. Maybe I should make it an upgrade (from robotics) instead of roundstart-available?

**Comment by ShadowDeath6** [comment]
no, making it an upgrade from robotics means they'll never get it

**Comment by Super3222** [comment]
Make it an upgrade. They shouldn't be give it off the bat. That's taking over 3 different jobs.

**Comment by ShadowDeath6** [comment]
any borg can do any job that uses consoles, apart from things requiring hands like engineering. 

Considering that it can only effectively do what its module is then it makes it less effective than a doctor with access to nanomed.

Unless we make the defib module have low resource cost and low research level nobody will EVER make it, just like nobody ever makes VTEC or the disabler cooler upgrade unless they're literally rolling in materals.

**Comment by Super3222** [comment]
It'll have low research levels, but you'll need to work for it. I don't want anyone slipping in a medical scanner to get these things.
